backend/tests/apps/common/management/commands/generate_model_graphs_test.py

```
@@ -0,0 +1,33 @@
+
+from django.conf import settings
+from django.core.management import call_command
+from django.test import SimpleTestCase
+
+
+class GenerateModelGraphsCommandTests(SimpleTestCase):
+    @patch("apps.common.management.commands.generate_model_graphs.call_command")
+    def test_command_runs_successfully(self, mock_call_command):
+        """
+        Command runs successfully and creates output directories.
+        """
```

Ruff (0.14.14)

[warning] One-line docstring should fit on one line

Reformat to one line

(D200)